### PR TITLE
Adding CI via github actions

### PR DESCRIPTION
**Pull request by esromneb**
This addresses issue #43

I've added builds for version 8-14. It seems like something in karma doesn't work in node 15 right now.

### CODE REVIEW

File: src/easystar.js [hunk 1]
```diff
@@ -159,6 +159,18 @@ EasyStar.js = function() {
...
 
+    /**
+    * Get the directional conditions previously set on a tile
+    *
+    * @param {Number} x The x value of the point.
+    * @param {Number} y The y value of the point.
+    * @return {Number} A list of all the allowed directions that can access
+    * the tile.
+    **/
+    this.getDirectionalCondition = function(x, y) {
```
Comment (prettymuchbryce) [inline src/easystar.js:170]: Did you intend to include this as part of this PR?

Comment (esromneb) [reply src/easystar.js:170]: No that was a mistake, my bad!

Comment (esromneb) [reply src/easystar.js:170]: But this is a feature missing from the library by the way

Comment (prettymuchbryce) [reply src/easystar.js:170]: Ah ok. We could consider adding it in another PR.

**Pull request closed without merging**